portal/src/main/webapp/js/src/study-view/data/StudyViewProxy.js

```diff
@@ -221,11 +221,11 @@ var StudyViewProxy = (function() {
                         var _parts = _lists[i].split('\t');
```

Comment:
@zheins thanks for the details. If you add these as comments to the code I think it is already an improvement. Another improvement would be to give an error message in case the length isn't 5 (since you indicate that in this case there is something wrong - or maybe we can add a strict validation to this, i.e. does it happen when one of these 5 fields is missing in DB record? ).